examples_tests/13.BRDF_Explorer/BRDFExplorerApp.cpp

```diff
@@ -506,29 +506,35 @@ void BRDFExplorerApp::eventAOTextureBrowse(const ::CEGUI::EventArgs&)
         const auto image = ext::cegui::loadImage(p.second.c_str());
         loadTextureSlot(ETEXTURE_SLOT::TEXTURE_AO, image.buffer, image.w, image.h);
 
+        char buffer[1024];
```

Comment:
Let's use tooltip as variable name instead of buffer like this change: https://github.com/manhnt9/IrrlichtBAW/pull/1/files/1c2ff50f43ecc04c8a51dd00a738eb636a4a71be..9ea3806325ca80e265b18185e0da7eea4af3e0b6#diff-060531d0748dffb86f3279465a244547R627

Reply:
Ouch... nice catch.